addons/map/functions/fnc_moduleMap.sqf

```diff
@@ -10,6 +10,8 @@
  */
 #include "script_component.hpp"
 
+if !(hasInterface) exitWith {};
```

Comment:
To work on a dedicated server this should be:
`if !(isServer) exitWith {};`

EFUNC(common,readSettingFromModule) only works when called from the server